jenkins-builds/6.0.UP02-2016.11-gpu

```diff
@@ -16,6 +16,8 @@
  h5py-2.6.0-CrayGNU-2016.11-Python-3.5.2-parallel.eb
  h5py-2.6.0-CrayGNU-2016.11-Python-2.7.12-serial.eb
  h5py-2.6.0-CrayGNU-2016.11-Python-3.5.2-serial.eb
+ influxdb-4.1.0-CrayGNU-2016.11-Python-2.7.12.eb
```

Comment:
Are users asking/going to use this package? if this is for internal CSCS usage I'd suggest we don't add it to the jenkins production list.